src/io/flutter/view/InspectorPerfTab.java

```diff
@@ -66,19 +66,22 @@
   }
 
   private void buildUI() {
-    setLayout(new GridLayout(1, 2, 5, 5));
+    setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
     setBorder(JBUI.Borders.empty(5));
 
     // Header
-    final JPanel headerPanel = new JPanel(new BorderLayout(0, 3));
-    headerPanel.add(new JBLabel("Running in " + app.getLaunchMode() + " mode"), BorderLayout.NORTH);
+    final JPanel headerPanel = new JPanel();
+    final BoxLayout boxLayout = new BoxLayout(headerPanel, BoxLayout.X_AXIS);
+    headerPanel.setLayout(boxLayout);
+    headerPanel.add(new JBLabel("Running in " + app.getLaunchMode() + " mode. "));
     if (app.getLaunchMode() == FlutterLaunchMode.DEBUG) {
       headerPanel.add(
-        new JBLabel("<html><body><p style='color:red'>Note: for best results, re-run in profile mode</p></body></html>"),
-        BorderLayout.SOUTH
-      );
+        new JLabel("<html><body><p style='color:red'>Note: for best results, re-run in profile mode</p></body></html>"));
```

Comment:
why did this go from JBLabel to JBLabel?
Generally we should use the JBContainerName classes instead of JContainerName classes to get a look and feel more consistent with intellij not that it matters in this case.

Reply:
Just an oversight. Reverted to JBLabel.